tools/util/cassandra_stress.py

```diff
@@ -0,0 +1,50 @@
+from contextlib import contextmanager
+import subprocess
+
+class CassandraStress:
+    def __init__(self, cassandra_dir):
+        self.stress_cmd = f"{cassandra_dir}/tools/bin/cassandra-stress"
+
+
+    def run(self, command, workload_size, node_ip, keyspace="ks", threads=1):
+        subprocess.run(args=[self.stress_cmd, 
+                             command, "no-warmup", "n={}".format(workload_size), 
+                             "-node", node_ip, 
+                             "-rate", "threads={}".format(threads), 
+                             "-schema", "keyspace={}".format(keyspace)],
+                             check=True)
+
+
+@contextmanager
+def cassandra_stress(cassandra_version="3.11.16"):
+    try:
+        # Fetch the cassandra
+        print(f"Fetching cassandra {cassandra_version}")
```

Comment:
Yes, I even considered to do so. I can add some CLI argument providing path the path to cassandra-stress tool.

I think we can follow the same logic we use during scylla docker container setup:
- if user specified scylla ip, we assume that the container is healthy, up and running (CI run case)
- otherwise, we set up the container ourselves (lazy repo user case)

We could do the same for fetching cassandra-stress. We can fetch cassandra in GH workflow, and then provide the path to cassandra-stress to test scripts. However, if someone wanted to run these scripts locally, we should fetch cassandra from python.